packages/main/src/controllers/download.ts

```diff
@@ -8,6 +8,7 @@ import { getTrackArtist, getTrackTitle } from '../utils/tracks';
 import Download from '../services/download';
 import Logger, { $mainLogger } from '../services/logger';
 import Window from '../services/window';
+import { platform } from 'os';
```

Comment:
Could you use this service for determining the platform?
https://github.com/nukeop/nuclear/blob/facaa4065024abd9f1786b10041d85b46a9dfd20/packages/main/src/services/platform/index.ts

Reply:
Of course. I have changed the code to use the platform service and pushed it to this branch. Did that also update the pull request?